### PR TITLE
Capabilities pane - disable authentication

### PR DESCRIPTION
**Pull request by micgibso**
Signed-off-by: micgibso <MICGIBSO@uk.ibm.com>

## What type of PR is this ? 

- [ ] Bug fix
- [ X] Enhancement

## What does this PR do ?
Describes the new Capabilities pane function to disable metrics.

## Which issue(s) does this PR fix ?
https://github.com/eclipse/codewind/issues/2712

## Does this PR require a documentation change ?
Yes

## Any special notes for your reviewer ?
Topic checked with Acrolinx.

### CODE REVIEW

**Comment by micgibso** [comment]
@markcor11 Mark, here's the PR for disabling metrics, please review, thx.

File: docs/_documentations/performance.md [hunk 1]
```diff
@@ -28,7 +28,20 @@ You will learn how to add and enable monitoring to your application. Then, you c
...
+
+![Performance dashboard capabilities](images/performanceguide/performance_dashboard_capabilities.png)
+
+- **Project status:** Provides current project status.
+- **Run Load Feature:** Provides information about whether load pressure can be applied to your project.
+- **Live monitoring:** Use this information to determine whether you can view live metrics. For example, Appsody projects protect the metrics endpoint by requiring authentication. Authentication can stop the performance dashboard from accessing the endpoint and from being able to retrieve performance metrics. You can programmatically resolve disabling authentication by following the suggestions that are provided, or you can click **Disable authentication** and Codewind disables authentication for you. When you click **Disable authentication**, you see that the project status updates and messages appear in the **Notification** pane. You can stack the notifications, remove them after a few seconds, or close them. 
+- **Benchmarks:** Benchmark your project as you run your tests. The charts display comparisons across load runs.
```
Comment (markcor11) [inline docs/_documentations/performance.md:42]: Exactly,   Benchmarks are a collection of completed load test results displayed on a chart and in a table.

File: docs/_documentations/performance.md [hunk 2]
```diff
@@ -28,7 +28,20 @@ You will learn how to add and enable monitoring to your application. Then, you c
 1. From your IDE, go to the Codewind plug-in and right-click on a project.
 2. Then, select **Performance Dashboard**. A window appears that displays the Performance Dashboard.<br>
 
-![performance dashboard](images/performanceguide/performancedash_window.png){:height="375px" width="800px"}<br>
+![Performance dashboard](images/performanceguide/performancedash_window.png){:height="375px" width="800px"}<br>
+
+## Viewing the status of your project
```
[the remark this answers is not in the document]
Comment (micgibso) [reply docs/_documentations/performance.md:33]: Amended.

File: docs/_documentations/performance.md [hunk 3]
```diff
@@ -28,7 +28,20 @@ You will learn how to add and enable monitoring to your application. Then, you c
...
+
+1\. View the status of your project, and the performance dashboard features available to it, in the **Capabilities** pane:
+
+![Performance dashboard capabilities](images/performanceguide/performance_dashboard_capabilities.png)
+
+- **Project status:** Provides current project status.
+- **Run Load Feature:** Provides information about whether load pressure can be applied to your project.
+- **Live monitoring:** Use this information to determine whether you can view live metrics. For example, Appsody projects protect the metrics endpoint by requiring authentication. Authentication can stop the performance dashboard from accessing the endpoint and from being able to retrieve performance metrics. You can programmatically resolve disabling authentication by following the suggestions that are provided, or you can click **Disable authentication** and Codewind disables authentication for you. When you click **Disable authentication**, you see that the project status updates and messages appear in the **Notification** pane. You can stack the notifications, remove them after a few seconds, or close them.
```
[the remark this answers is not in the document]
Comment (micgibso) [reply docs/_documentations/performance.md:41]: Redrafted, but please recheck.

File: docs/_documentations/performance.md (hunk 1 above)
Comment (micgibso) [reply docs/_documentations/performance.md:42]: Changed to your suggestion.

**Comment by micgibso** [comment]
@markcor11 Review comments incorporated but please recheck. Appreciate the graphic quality isn't very good/out of date, could you grab an updated screen shot and paste it in here? I'll then update.  Thank you.

**Comment by micgibso** [comment]
I've renamed the screen `Project Capabilities Overview` and updated it all accordingly. Considered putting the icon switch at the top but the first thing you want to know about is the content of the screen so left it at the bottom of the section.  Comments appreciated.

**Comment by micgibso** [comment]
@sishida Please add approving review. thx.

**Pull request merged**